pkg/apis/monitoring/v1/types.go

```diff
@@ -285,6 +285,10 @@ type PrometheusSpec struct {
 	// based on a service monitor can access arbitrary files on the file system
 	// of the Prometheus container e.g. bearer token files.
 	ArbitraryFSAccessThroughSMs ArbitraryFSAccessThroughSMsConfig `json:"arbitraryFSAccessThroughSMs,omitempty"`
+	// OverrideHonorLabels if set to true overrides all user configured honor_labels
+	// if set in ServiceMonitor or PodMonitor to true, this flag will take precedence
```

Comment:
honor labels should be enforced to be `false`

Reply:
sure, I see your point can you suggest a better description?

Reply:
just change the second `true` to `false` no?

Reply:
As in

```
OverrideHonorLabels if set to true overrides all user configured honor_labels
if set in ServiceMonitor or PodMonitor to false.
```

Reply:
> OverrideHonorLabels if set to true overrides all user configured honor_labels
if set in ServiceMonitor or PodMonitor to false.


Maybe:

> OverrideHonorLabels if set to true overrides all user-configured honor_labels. If set in ServiceMonitor or PodMonitor it overrides `honor_labels` to false.

WDYT?